exampleCourse/courseInstances/Sp15/assessments/hw04-demoQuestions/infoAssessment.json

```diff
@@ -48,7 +48,8 @@
                 {"id": "demoRandomPlot",                      "points": 1, "maxPoints": 3},
                 {"id": "demoRandomDataFrame",                 "points": 2, "maxPoints": 6},
                 {"id": "demoAnsiOutput",                      "points": 2, "maxPoints": 4},
-                {"id": "demoOverlayTriangle",                 "points": 2, "maxPoints": 4}
+                {"id": "demoOverlayTriangle",                 "points": 2, "maxPoints": 4},
+                {"id": "demoExtendableElement",               "points": 2, "maxPoints": 4}
```

Comment:
Would it be possible to show how to extend an existing `pl-*` element? The `<extendable-element>` demo is useful, but feels too detached from practical usage.

Reply:
See below comment, but existing elements will need to add support for extensions to be able to use them.